MinnPost/object-sync-for-salesforce
# activate db version fix

## What does this PR do?

This fixes #259, ensuring that if the database activate method runs, the current installed version option is updated.

## How do I test this PR?

- Run the plugin with an out of date database and make sure it gets updated.
- Run it with an up to date database and nothing should happen.
